doc/release/README.chplenv

```diff
@@ -238,14 +238,17 @@ Optional Settings
    options are:
 
    CHPL_TASKS:
-        qthreads       : run tasks using Sandia's Qthreads package
-        fifo           : run tasks to completion, in FIFO order
-        massivethreads : run tasks using U Tokyo's MassiveThreads package
+        qthreads       : use Sandia's Qthreads package
+        fifo           : use POSIX threads
+        massivethreads : use U Tokyo's MassiveThreads package
+        muxed          : use Cray-specific lightweight threading
```

Comment:
or if it doesn't look too ugly you could be something like  "(prebuilt module only)" after the description